internal/email/message.go

```diff
@@ -10,11 +10,11 @@ import (
 )
 
 type Message struct {
-	To      []string `json:"to"`
+	To      []string `json:"to" required:""`
 	CC      []string `json:"cc"`
 	BCC     []string `json:"bcc"`
-	Subject string   `json:"subject"`
-	Body    string   `json:"body"`
+	Subject string   `json:"subject" required:""`
+	Body    string   `json:"body" required:""`
```

Comment:
You can send messages without a body.
Especially before most common chat platforms like Slack,
people would sent emails that were only subjects and end with "EOM" ("End of Message").